planet/cli/subscriptions.py

```diff
@@ -169,7 +162,8 @@ async def list_subscription_results_cmd(ctx,
                                         status,
                                         limit):
     """Gets results of a subscription and prints the API response."""
-    async with subscriptions_client(ctx) as client:
+    async with Session(auth=ctx.obj['AUTH']) as session:
+        client = SubscriptionsClient(session)
```

Comment:
Changing our usage pattern adds 6 lines of code (one per command) and removes 12 lines of code (the subscriptions_client, its requirements, and docs).